docs/source/testing.rst

```diff
@@ -97,41 +97,62 @@ Smoke test:
     $ cd koku/testing
     $ ./run_smoke_tests.sh
     
+=====================
+PDB in Docker
+=====================
 To use pdb while running the koku-server in docker:
-Ensure all migrations are run.
-Stop the server `docker-compose stop koku-server`
-Run the server with service-ports: `docker-compose run —service-ports koku-server`
-Breakpoints will now be stopped in this terminal window.
+    1. Ensure all migrations are run.
+    2. Stop the server `docker-compose stop koku-server`
+    3. Run the server with service-ports: `docker-compose run —service-ports koku- 
+       server`
+
+Breakpoints should now be stopped at, in this terminal window.
+
+To test a specific file using tox, edit this line, for example::
```

Comment:
Should we move the `PDB in Docker` section below these points about using tox but above the `PDB in IQE` section? I'm not sure if it is related - I could be misunderstanding.

Reply:
@abaiken 

I think I misnamed this section,  PDB in Docker is really: "Running PDB while Koku is deployed in Docker-compose" 

And "PDB in IQE" is really "Running PDB in the IQE docker container"
I can rename those

Reply:
@nbon12 My comment was unclear - sorry!  I was more so asking if the section about tox belonged underneath the header `PDB in Docker` where it currently is. If the tox documentation wasn't related to PDB in Docker - I was suggesting moving the entire PDB section to be moved below it or perhaps we could move the tox documentation & the `Unit testing log messages` sections to be somewhere underneath the unit tests section [here](https://github.com/project-koku/koku/blob/f38de9de5fd72b201bef855e63a04ae749dcb8df/docs/source/testing.rst#unit-testing).

Reply:
I think I agree with the direction @abaiken is going here. Let's look at the organization of the document sections and try to make sure it flows in a coherent way.

For example, the sections and sub-sections might be best organized like this:

- Debugging Options
  - Running PDB with docker-compose
  - Running PDB with IQE container
  - Running PDB inside openshift/crc
  - Alternative debugging ideas (e.g. Printing additional output, strategically placed sys.exit()s, etc.)

This way, we collect related information under the same heading so that it's easier to locate the information a reader wants to look at.